doc/ref_model/chapters/chapter04.md

```diff
@@ -265,28 +265,28 @@ Table 4-10 shows performance measurement capabilities.
+The following table of requirements are derived from the VNF/CNF applications, Cloud Infrastructure and Telco Operators needs to have multiple realisations of HW Acceleration and Offload functions that can also be implemented through SW when no special hardware is available. These requirements should be adopted in Reference Architectures to ensure that the different implementations on the market are as aligned as possible in their interfaces and that HW Acceleration and Offload functions get an efficient ecosystem of accelerators that compete on their technical merits and not through obscure or proprietary interfaces.
 
 **Table 4-12** shows Acceleration/Offload API Capabilities.
 
 <a name="Table4-12"></a>
 
 | Ref       | Acceleration/Offload API Capability | Unit   | Definition/Notes                                                                            |
 |-----------|--------------------------------------------|--------|---------------------------------------------------------------------------------------------|
-| e.api.001 | VNF/CNF usage of Accelerator standard i/f                           | Yes/No | VNF/CNF shall use abstracted standardized interfaces to the Acceleration/Offload function that enables multiple HW and SW implementation of the accelerated/offloaded function from multiple vendors in the Cloud Infrastructure                        |
-| e.api.002 | Virtualization Infrastructure SW usage of Accelerator standard i/f  | Yes/No | Virtualization Infrastructure SW shall use abstracted standardized interfaces to the HW-Acceleration/Offload  function that enables multiple HW and SW implementation in the HW Infrastructure Layer of the accelerated function from multiple vendors  |
-| e.api.003 | Accelerators offering standard i/f to HW Infra Layer                | Yes/No | Acceleration/Offload functions shall offer abstracted standardized interfaces for the Virtualization Infrastructure and VNF/CNF applications                                                                                                             |
-| e.api.004 | Accelerators offering virtualized functions                         | Yes/No | Acceleration/Offload functions for VNFs/CNFs should preferably be virtualized to allow multiple VNFs/CNFs to use the same Acceleration/Offload instance                                                                                                 |
+| e.api.001 | VNF/CNF usage of Accelerator standard i/f                           | Yes/No | VNF/CNF shall use abstracted standardised interfaces to the Acceleration/Offload function that enables HW and SW implementations of the accelerated/offloaded functions from multiple vendors in the Cloud Infrastructure.                        |
```

Comment:
```suggestion
| e.api.001 | VNF/CNF usage of Accelerator standard i/f                           | Yes/No | VNF/CNF shall use abstracted standardised interfaces to the Acceleration/Offload functions. This would enable use of HW and SW implementations of the accelerated/offloaded functions from multiple vendors in the Cloud Infrastructure.                        |
```